<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: grey
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="nameLine">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
color:white
</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ageBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
color:white
</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="birthdayLine">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
color:white
</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>0</y>
      <width>20</width>
      <height>441</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="serializeBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
color:white
</string>
    </property>
    <property name="text">
     <string>Serialize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deserializeBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
color:white
</string>
    </property>
    <property name="text">
     <string>Deserialize</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
